velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -634,6 +634,25 @@ class PlanBuilder {
       const std::vector<std::string>& aggregationInputs,
       std::string groupIdName = "group_id");
 
+  /// Add an ExpandNode using specified projections. See comments for
+  /// ExpandNode class for description of this plan node.
+  ///
+  /// @param projections A list of projection expressions. Each expression is
+  /// either a column name, null or non-null constant.
+  ///
+  /// For example,
+  ///
+  ///     .expand(
+  ///            {{"k1", "k2", "a", "b", "0 as group_id_0"}, // Column name will
+  ///            be extracted from the first projection.
+  ///             {"k1", "null", "a", "b", "1"},
+  ///             {"null", "null", "a", "b", "2"}})
+  ///
+  ///
+  PlanBuilder& expand(
+      const std::vector<std::vector<std::string>>& projections,
+      TypePtr groupIdColumnType = BIGINT());
```

Comment:
We have discussed before that Expand node/operator should not be tightly coupled with 'group ID'. Hence, groupIdColumnType parameter here is unexpected. Also, it is not documented. Would you clarify what are you trying to do? Can you find a way to not couple this API with 'group ID'?

Reply:
@mbasmanova The purpose of `groupIdColumnType `  is to set the type for non-null constant columns [here](https://github.com/facebookincubator/velox/pull/5403/files#diff-fd8dcf1389140cd0b653207240aa59d35e7959de82ed4e08369b78a85737f04fR980). How about renaming `groupIdColumnType ` to `nonNullConstantType ` instead?

Reply:
Why is this necessary? Can we not use core::ConstantExpr::type() ?

Reply:
@mbasmanova Yes, we can use core::ConstantExpr::type() to set it. Updated.